components/table/Table.jsx

```diff
@@ -92,59 +92,58 @@ const Table = (props) => {
     return (
         <>
             {props.isStacked && props.updateSort && (
-            <div className="usa-dt-table__stacked-picker">
-                <label htmlFor="stackedTableSort">Sort By</label>
-                <Picker
-                    id="stackedTableSort"
-                    selectedOption={props.currentSort.field}
-                    options={union(getTablePickerOptionsAsc, getTablePickerOptionsDesc)} />
-            </div>
-        )}
-            <table className={`usda-table ${stackedClass} ${props.classNames}`}>
-            {props.screenReaderCaption && (
-                <caption className="usa-dt-sr-only">{props.screenReaderCaption}</caption>
+                <div className="usa-dt-table__stacked-picker">
```

Comment:
should this change for the Table component be in this PR?

Reply:
yes, these changes are only for whitespace, and one suppression of a linting error for label elements having associated control elements.
so this was intentional.